docs/main/05-reference/02-resource-requests/01-deleting-a-resource-request.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Deleting a Resource Request
+title: Deleting
+---
+
+To delete a Resource Request, run:
+
+```
+kubectl delete <Promise CRD> <Resource Request Name>
+```
+
+Replace `<Promise CRD>` with the targeted Promise and `<Resource Request Name>` with the
+Resource Request you want to delete.
+
+To find the Promise CRD, run:
+
+```
+kubectl get crds
+```
+
+To find the Resource Request Name, run:
+
+```
+kubectl get <Promise CRD>
+```
+
+After the Resource Request is deleted it takes some time for Flux to
```

Comment:
Do we want to say Flux here? I think we have talked about staying GitOps tool agnostic.

Reply:
Thanks! changed to GitOps. I considered copying the same GitOps explainer statement at the end of the promise deletion to here too, but wansn't sure if its worth duplicating in both places. WDYT?

Reply:
IMO that is a good idea and it is worth it as they are independent pages rather than a guarantee (or even a suggestion) of reading order.

Reply:
sounds good, I'll update so both sections have the paragraph!